### ISSUE TITLE
travis: build/temp.linux-x86_64-2.7/_openssl.c:434:30: fatal error: openssl/opensslv.h: No such file or directory

### ISSUE THREAD

**Issue by dimi309** [conan-io/conan#1337]
Hi everyone,

I've noticed that today, a lot of my builds here:

https://travis-ci.org/dimi309/conan-packages/builds/237804359?utm_source=github_status&utm_medium=notification

fail, starting with this error:

```
building '_openssl' extension
    creating build/temp.linux-x86_64-2.7/build
    creating build/temp.linux-x86_64-2.7/build/temp.linux-x86_64-2.7
    x86_64-linux-gnu-gcc -pthread -fno-strict-aliasing -DNDEBUG -g -fwrapv -O2 -Wall -Wstrict-prototypes -fPIC -I/usr/include/python2.7 -c build/temp.linux-x86_64-2.7/_openssl.c -o build/temp.linux-x86_64-2.7/build/temp.linux-x86_64-2.7/_openssl.o
    build/temp.linux-x86_64-2.7/_openssl.c:434:30: fatal error: openssl/opensslv.h: No such file or directory
     #include <openssl/opensslv.h>
                                  ^
    compilation terminated.
    error: command 'x86_64-linux-gnu-gcc' failed with exit status 1
```

Is there something that has changed on travis? Am I making a mistake somewhere?

Btw, ignore the macos build. That is another issue but I will report it only after I have checked to make sure that I have not done something wrong in my package configuration.

Thanks!

**Comment by lasote** [comment]
Hi @dimi309, yes I updated yesterday the docker images with a newer cmake and installing ninja build, I don't know why it affects but I'm checking it right now. Thanks for reporting.

**Comment by lasote** [comment]
@dimi309 Please, restart your builds, I've uploaded the docker images with a couple of dependencies, ``libffi-dev`` and ``libssl-dev`` apparently needed by package tools. I've also updated the tests to ensure that the packages are able to install conan and conan-package-tools correctly.

**Comment by dimi309** [comment]
Hi @lasote 

All Linux builds work now. Thanks a lot!

**Comment by dimi309** [comment]
Hi @lasote 
This morning I started getting this error on travis, in my MacOS builds this time:

https://travis-ci.org/dimi309/conan-packages/jobs/238188982

It looks kind of similar to the Linux problem that was resolved yesterday, so I thought to report it here.

Is it indeed more or less the same situation?

**Comment by lasote** [comment]
Hi @dimi309 . The error was reported some days ago in py3 but now it affects to py2 too. https://github.com/conan-io/conan-package-tools/issues/68
We are going to release 0.23.0 today including this patch.

**Comment by dimi309** [comment]
Great! I'll upgrade and see how it goes. Thanks a lot!

**Comment by dimi309** [comment]
This has been resolved for quite some time. Sorry I forgot to report back.
